app/blueprints/plage_ouverture_blueprint.rb

```diff
@@ -0,0 +1,19 @@
+class PlageOuvertureBlueprint < Blueprinter::Base
+  identifier :id
+
+  fields :ical_uid, :title, :first_day, :start_time, :end_time
+  association :agent, blueprint: AgentBlueprint
+  association :organisation, blueprint: OrganisationBlueprint
+  association :lieu, blueprint: LieuBlueprint
+  association :motifs, blueprint: MotifBlueprint
+
+  field :rrule do |plage_ouverture|
+    ics = PlageOuverture::Ics.new(plage_ouverture: plage_ouverture)
+    ics.rrule
```

Comment:
pourquoi exposer `rrule` comme un champ a part ? c'est redondant car il est deja contenu dans le `ical`

Reply:
Pour éviter d'imposer l'installation d'un parser de iCal si seulement un parser de RRULE est nécessaire.

Reply:
👌 got it didn't know it was a thing